### ISSUE TITLE
input buttons issue

### ISSUE THREAD

**Issue by ozshabat** [OpenXbox/xbox-smartglass-nano-python#12]
Hey,

I've opened the Input Channel and Feedback Channel and tried to send an input button in a game.

I got this in the debug:

```
frame_id = 2371469396
    timestamp = 19910
    created_ts = 1585050122
    buttons = None
    analog = None
    extension = None
    byte_6 = 1
    dpad_up = 0
    dpad_down = 0
    dpad_left = 0
    dpad_right = 0
    start = 0
    back = 0
    left_thumbstick = 0
    right_thumbstick = 0
    left_shoulder = 0
    right_shoulder = 0
    guide = 0
    unknown = 0
    a = 0
    b = 0
    x = 0
    y = 4
    left_trigger = 0
    right_trigger = 0
    left_thumb_x = 0
    left_thumb_y = 0
    right_thumb_x = 0
    right_thumb_y = 0
    rumble_trigger_l = 0
    rumble_trigger_r = 0
    rumble_handle_l = 0
    rumble_handle_r = 0
```

Any idea why the input channels dont work?

**Comment by tuxuser** [comment]
Fixed with PR #13 

There were several flaws:
* message.header did not take `**kwargs`, hence failing when passed `sequence_num`
* Timestamping was not correct - while at it, replaced `ReferenceTimestamp` with a `datetime`-implementation (construct.Adapter -> `ReferenceTimestampAdapter`)
* Field type for thumbstick positions were incorrect (`input_frame_analog`).
  Changed from `uint16` BE to `int16 LE`

In client.py replace 
https://github.com/OpenXbox/xbox-smartglass-nano-python/blob/0dbd86ce6ee15d0056d9dc93c1984d82f6f25272/xbox/nano/scripts/client.py#L53

with
```py
client = SDLClient(1280, 720, use_keyboard=True)
```

to test quickly.

**Comment by osfunapps** [comment]
It works great! this issue can be closed
